.github/workflows/ci.yml

```diff
@@ -1,15 +0,0 @@
-on: [push, pull_request]
-name: CI
-jobs:
-  test:
-    name: Test
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Init Hermit
-        run: ./bin/hermit env --raw >> $GITHUB_ENV
-      - name: Test
-        run: go test ./...
-      - name: Lint
-        run: golangci-lint run
```

Comment:
FWIW, and I'm biased here, but you should check out Hermit - it ensures that developer machines and CI machines are both using identical tooling. Not so important with Go, but very important with golangci-lint.

Also you've replaced 15 lines of config with 50 linues of config, so not sure I'd call it "simplifying" ;)

Reply:
Sorry, I hadn't realised the nuance in the source of Hermit there :-)

My main thought process was around using the `golangci/golangci-lint-action@v2` action which automatically adds lint errors to the code inside github and is very easy to incorporate. I'm not sure if Hermit supports this. The simplification part came from removing the `bin` directory which could potentially be confusing as it is not useful to end users of the library; my immediate thought was "why is there in a bin directory for a package?".

I've not had any problems with `golangci-lint`. For most of our projects however we use Docker containers which in my experience is more reliable than playing with environment variables like Hermit appears to do. I also develop 100% on remote Linux boxes, so don't run into performance issues with docker running on OS X.

My personal take: Hermit looks useful for private or complex projects, but for an open library, I think there is value in sticking to "standard" procedures.  If you feel really strongly about this however, I don't mind reverting.